### ISSUE TITLE
Display error message in tap results

### ISSUE THREAD

**Issue by ltegman** [hapijs/lab#761]
On our CI server we print test results in tap format which is picked up by the CI's web UI, but we've been finding that the tap results from lab are not very helpful, because if there's no stack track it won't print anything in the YAML block other than the time in ms.

Would you accept a PR that will print out a message field in the YAML block even if there is no stack trace?

**Comment by geek** [comment]
@ltegman what will it print in the message field when there isn't a stack trace?

**Comment by ltegman** [comment]
In the cases we ran across where there was no stack trace the `test.err` property existed, but it was simply a string with the error message in it. I was thinking we could check if the `test.err` property is a string instead of an object and if so simply print that string as the message.

**Comment by jtlapp** [comment]
I was planning to add this to lab once I got more comfortable with the tool -- I only just started using it. It's one thing I miss from tap. One issue with stack traces in test suites is that you get lots of unwanted noise from the test runner's call stack, as well from supporting libraries. I handled this in [`subtap`](https://github.com/jtlapp/subtap) by allowing the user to specify points of truncation within the call stack using a [`SUBTAP_UNSTACK_PATHS`](https://github.com/jtlapp/subtap#subtap_unstack_paths) environment variable.
